### DISCUSSION TITLE
Extension's latest version stuck in verification phase

### DISCUSSION THREAD

**Discussion by felix8** [microsoft/vscode-discussions#2402] [Extension Development QnA] [answered]
Hey VS Code team,

My extension [Mode's](https://marketplace.visualstudio.com/items?itemName=aruna-labs.mode) latest version has been stuck in the verification process for two days. How can I unblock its release? We have a major launch coming up, and this version is crucial to showcasing what Mode can do. Thank you so much—I really appreciate your help!

Publisher: aruna-labs (my org)
Extension: Mode
Stuck version: 1.0.7

**Comment by arunalabs** [comment, marked as answer]
The version was published exactly around 48 hours and we're now unblocked. Feel free to resolve this issue.
